plugins/user/cronjobs

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+# -*- python -*-
+# -*- coding: utf-8 -*-
+#
+# Plugin to monitor the number of cronjobs running per user
+#
+# Usage: Place in /etc/munin/plugins/ (or link it there using ln -s)
+#   Add this to your /etc/munin/plugin-conf.d/munin-node:
+#       [cronjobs]
+#       user root
+#
+# 2019-09-09: v 1.0 pcy <pcy.ulyssis.org>: created
+#
+# Parameters understood:
+#
+#   config   (required)
+#   autoconf (optional - used by munin-config)
+#
+# magic markers:
+#%# family=auto
+#%# capabilities=autoconf
+
+from datetime import datetime, timedelta
+import shutil
+import sys
+
+
+def isokline(ln: str):
+    if "CRON" not in ln:
+        return None
+    sp = ln.split()
+    return sp if sp[5][0] == '(' and sp[5][-1] == ')' and sp[5] != "(root)" else None
+
+
+def getfilelines():
+    with open('/var/log/syslog', 'r') as f:
+        return [x for x in map(isokline, f.readlines()) if x is not None]
+
+
+def autoconf():
+    print("no (need cron installed)" if shutil.which('cron') is None else "yes")
+
+
+def config():
+    usernames = set(map(lambda e: e[5][1:-1], getfilelines()))
```

Comment:
Maybe move the interpretation of the lines into `getfilelines` (and rename it), in order to create a simpler interface?

Reply:
Done (reworked it a little bit). Can't exactly move the username filtering to (what was) `getfilelines`, as I also use it in `config` for checking timestamps as well.